pages/common/bedops.md

```diff
@@ -0,0 +1,24 @@
+# bedops
+
+> BEDOPS: A fast, highly scalable, and easily-parallelizable genome analysis toolkit.
+> Apply efficient set operations on any number of sorted BED or Starch files such as element-of, intersection, merge, difference, partition, etc. and their logical inverses.
+
+- Take the multiset union of two or more BED or Starch files and write the sorted result to {{path/to/output_file}}:
```

Comment:
We can just mention "output_file" like this. The full path is more suited in the actual example.

Reply:
That's fine.